src/ensembl/src/shared/components/loading-button/ControlledLoadingButton.tsx

```diff
@@ -27,24 +27,32 @@ import { LoadingState } from 'src/shared/types/loading-state';
 
 import styles from './LoadingButton.scss';
 
-type Props = {
+export type Props = {
```

Comment:
Why do we need to export this here?

Reply:
I was trying to derive the props of the `LoadingButton` from the props of `ControlledLoadingButton`, but then gave up on the idea. This was the remnant of that attempt. I've removed the export now.